packages/adapter-squid/test/integration/DependencyRegistrar.integration.test.ts

```diff
@@ -1,20 +1,38 @@
 import "reflect-metadata";
+import * as path from "path";
+import pkg from "../../package.json";
 import { container } from "tsyringe";
-import { IValidator } from "@mate/sdk";
-import { DependencyRegistrar } from "../../src/DependencyRegistrar";
-import { ISquidProvider, SquidProvider } from "../../src/services";
-import { SquidStep } from "../../src/types";
-import { StepConfigValidator } from "../../src/validation";
+import {
+   IValidator,
+   DependencyRegistrar as SdkDependencyRegistrar,
+} from "@mate/sdk";
+import { DependencyRegistrar as SquidDependencyRegistrar } from "../../src/DependencyRegistrar";
+import {
+   ISignerService,
+   ISquidProvider,
+   SignerService,
+   SquidProvider,
+} from "../../src/services";
+import { SquidAdapterConfig, SquidStepConfig } from "../../src/types";
+import {
+   AdapterConfigValidator,
+   StepConfigValidator,
+} from "../../src/validation";
+import { SquidAdapter } from "../../src/SquidAdapter";
 
 describe("DependencyRegistrar", () => {
-   let registrar: DependencyRegistrar;
-
    const baseUrl = "https://testnet.api.squidrouter.com";
    const integratorId = "mate-sdk";
 
    beforeAll(() => {
-      registrar = DependencyRegistrar.getInstance();
-      registrar.configure();
+      // Override the config file path to point to the exapmle config
+      process.env["CONFIG_PATH"] = path.resolve(
+         __dirname,
```

Comment:
i think `__dirname` is on the way to being deprecated => https://blog.logrocket.com/alternatives-dirname-node-js-es-modules/

hate the new syntax but apparently that's the new way to ensure future es-module compatibility